### PR TITLE
Add a new detailed Flatpak guide for beginner users to Silverblue.

### PR DESCRIPTION
**Pull request by mikemorrell2**
I made a new Flatpak guide that is a detailed, hands-on/end-to-end guide that shows how to setup Flathub and install new Flatpak apps. It is more verbose with many screenshots and is geared towards new users. Please review for content accuracy, spelling, grammar, improvement ideas, etc... I know it does not contain any command line Flatpak instructions yet and maybe that should be added at the end as additional content. I would like to get it pulled in as-is so that it can be useful now and I can add the command line content later.

### CODE REVIEW

File: modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -3,6 +3,7 @@
 * xref:first-login.adoc[First Login]
 * xref:upgrades.adoc[Upgrades and Rollbacks]
 * xref:silverblue-guide.adoc[Silverblue Guide]
+* xref:silverblue-flatpak-guide.adoc[Silverblue Flatpak Guide]
```
Comment (matthiasclasen) [inline modules/ROOT/nav.adoc]: As a general comment, I don't think it is right to accumulate multiple comprehensive user guides next to each other - these should be merged and made to fit into the overall documentation.

**Comment by matthiasclasen** [comment]
I'll do a review when I have more time

File: modules/ROOT/nav.adoc (hunk 1 above)
Comment (mikemorrell2) [reply modules/ROOT/nav.adoc]: Okay, I merged flatpak guide into the overall silverblue guide and removed the separate file. Should be good to go now.

File: modules/ROOT/pages/silverblue-guide.adoc [hunk 2]
```diff
@@ -159,6 +159,136 @@ there have likely noted when they did a remote-ls command on their configured
 flathub remote, there are more things listed than the app's. There are runtimes,
 and sdk's as well as app's since Flathub.org is a place to distribute such things.
 
+==== Installing Flatpak Applications
+===== Gnome Software
+Gnome Software is the default graphical software installation application that
```
Comment (matthiasclasen) [inline modules/ROOT/pages/silverblue-guide.adoc]: Please capitalize GNOME, throughout.

File: modules/ROOT/pages/silverblue-guide.adoc [hunk 3]
```diff
@@ -159,6 +159,136 @@ there have likely noted when they did a remote-ls command on their configured
...
+
+After clicking the `Install` button, a download information window will be shown.
+Verify the correct Flatpak is being downloaded and then click on the `OK` button
+to begin installing the LibreOffice application.
+
+image::sfg_libreoffice_install.png[title="LibreOffice Flatpak download"]
+
+Once the Flatpak is downloaded, the Gnome Software application will open a new
+window with an `Install` button (1). This window also shows details again about
+the Flatpak application (2). Notice that LibreOffice is Sandboxed. This means it
```
Comment (matthiasclasen) [inline modules/ROOT/pages/silverblue-guide.adoc]: 'sandboxed' shouldn't be capitalized here.

File: modules/ROOT/pages/silverblue-guide.adoc [hunk 4]
```diff
@@ -159,6 +159,136 @@ there have likely noted when they did a remote-ls command on their configured
...
+will have limited interaction with other application and the operating system for
+improved security. Also, there is no localization of languages for this application,
+so you may need to install additional extensions to provide support for your language.
```
Comment (matthiasclasen) [inline modules/ROOT/pages/silverblue-guide.adoc]: locale extensions are installed automatically

File: modules/ROOT/pages/silverblue-guide.adoc (hunk 2 above)
Comment (mikemorrell2) [reply modules/ROOT/pages/silverblue-guide.adoc]: Okay. I changed all cases of Gnome to GNOME.

File: modules/ROOT/pages/silverblue-guide.adoc (hunk 3 above)
Comment (mikemorrell2) [reply modules/ROOT/pages/silverblue-guide.adoc]: Fixed capitalization of sandboxed.

File: modules/ROOT/pages/silverblue-guide.adoc (hunk 4 above)
Comment (mikemorrell2) [reply modules/ROOT/pages/silverblue-guide.adoc]: I removed the sentence about localization.

**Comment by mikemorrell2** [comment]
Okay, I resolved all the open comments. Anymore?

**Pull request merged**